packages/devextreme/js/__internal/ui/tree_view/m_tree_view.base.ts

```diff
@@ -542,22 +619,22 @@ class TreeViewBase extends HierarchicalCollectionWidget<TreeViewBaseProperties>
     const items = extend(true, [], this.option('items'));
```

Comment:
We make deep copy of array, then go through it and remove from original array all items that are null or undefined. Isn't it the same as
```suggestion
    let { items } = this.option();
    items = items.filter((item) => !item);
```